front/components/vaults/VaultLayout.tsx

```diff
@@ -45,3 +79,85 @@ export function VaultLayout({
     </RootLayout>
   );
 }
+
+function VaultBreadCrumbs({
+  owner,
+  vault,
+  category,
+  dataSource,
```

Comment:
We never exposes dataSource in this part so I guess it can be removed.

Reply:
cf comment below, not sure how to do without

Reply:
Thought I had deleted the ref; thanks for noticing :pray: